app/components/event/_event.html.erb

```diff
@@ -28,6 +28,12 @@
         <span class="icon-font--date"></span>
         <%= date %>
       </div>
+      <% if online? %>
```

Comment:
This is good but could we have the URL here?

Reply:
We discussed and it's handled on the partner view page in the page description.